netjsonconfig/backends/base/backend.py

```diff
@@ -127,7 +127,7 @@ def render(self, files=True):
         if self.intermediate_data is None:
             self.to_intermediate()
         # support multiple renderers
-        renderers = getattr(self, 'renderers', [self.renderer])
+        renderers = getattr(self, 'renderers', None) or [self.renderer]
```

Comment:
have you rebased on master yet? This change shouldn't show up because is already on master